src/app.vue

```diff
@@ -86,11 +88,8 @@ watch(accountErrors.value, (val) => { if (val && val.length > 0) { handleError(v
           <bcros-contact-info class="font-normal font-16 mt-4" :contacts="getContactInfo('registries')" />
         </template>
       </bcros-dialog>
-      <div v-if="appLoading">
-        <UIcon
-          name="i-heroicons-arrow-path"
-          class="animate-spin text-[50px] text-gray-700 absolute top-40 left-[50%]"
-        />
+      <div v-if="dashboardIsLoading || appLoading">
+        <BcrosLoadingIcon class="animate-spin text-6xl text-gray-700 absolute top-40 left-[50%]" />
```

Comment:
Do you think we could move this to the layout? That way it could include covering the business details header. I think it should work pretty good if we replace the 'appLoading' with your new 'trackUiLoadingStart/Stop' inside the onMounted in here

Reply:
moved them to the layouts, and replaced main loading with track loading stuff.